ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -129,6 +102,8 @@ open class TwoLineTitleView: UIView {
         return interactivePart == .title ? accessoryType : .none
     }
 
+    private var currentStyle: Style
```

Comment:
I can see in the screenshots that switching to dark mode seems to work, but are we certain this will work outside of our weird demo controller dark mode override?

Reply:
Actually... Did you use the colour contrast analyser to ensure the colors are actually changing between dark and light mode?